### ISSUE TITLE
Could you please upload your parsed label for SYNTHIA dataset? as wI cannot reproduce the result for SYN to Cityscape

### ISSUE THREAD

**Issue by changliu816** [liyunsheng13/BDL#3]

**Comment by liyunsheng13** [comment]
I upload the labels to Google Drive. You can download them with the following link [synthia_mapped_to_cityscapes](https://drive.google.com/open?id=1ShCoaYMP1G5U3fLPCvfedQ-BFGsA2zK4)

**Comment by changliu816** [comment]
Thank you !

**Comment by changliu816** [comment]
Sorry to interrupt again, the link seems broken

**Comment by liyunsheng13** [comment]
Just update the link. Sorry about it.

**Comment by changliu816** [comment]
It works now. Thank you !

**Comment by changliu816** [comment]
Hi yunsheng, 
For SYNTHIA to Cityscapes experiment, I found that the common categories between these two dataset are 16 compared to GTA5->Cityscape that has 19 categories. My question is that did you train SYNTHIA to Cityscapes with 16 categories or 19 categories? Because in SYNTHIA data loader, the category mapping contains 19 common classes but your paper stated that only 16 categories are overlapped in training which confused me. 

I trained SYNTHIA to Cityscapes with 19 categories without SSL and BDL as shown in your SYNTHIA data loader, but got following result 
===> mIoU19: 19.76
 ===> mIoU16: 23.47
===> mIoU13: 27.49

Without SSL and BDL, the framework should be same as AdapSegNet whose result on SYNTHIA to Cityscapes is 37.6 (mIoU13)

**Comment by liyunsheng13** [comment]
It is a little weird. It should not be that bad. Could you share my the learning parameters you use? if you use my code, you can get the file to store all the parameters.

**Comment by changliu816** [comment]
Actually, I was running the code from here https://github.com/wasidennis/AdaptSegNet with your data loader and parsed label. Yes, I did not expect the result this bad. 
I would definitely try your code on SYNTHIA to Cityscapes to see what happens, and give you update in tomorrow. So, you trained SYNTHIA to Cityscapes with 19 classes right ?

**Comment by liyunsheng13** [comment]
I train with all classes but only report 16 or 13 in the paper.

**Comment by changliu816** [comment]
I see. Thanks for your considerate response !

**Comment by changliu816** [comment]
Hi Yunsheng, I trained SYNTHIA to Cityscapes with your code, but still got bad result. The parameters are shown below: 
----------------- Options ---------------
               batch_size: 1                             
                 data_dir: ../AdaptSegNet/data/RAND_CITYSCAPES
          data_dir_target: ../AdaptSegNet/data/Cityscapes/data
 data_label_folder_target: None                          
                data_list: ./dataset/synthia_list/train.txt
         data_list_target: ./dataset/cityscapes_list/train.txt
             init_weights: ./model/vggfcn_syn_init.pth   
        lambda_adv_target: 0.001                         
            learning_rate: 0.00025                       
          learning_rate_D: 0.0001                        
                    model: VGG                           
                 momentum: 0.9                           
              num_classes: 19                            
                num_steps: 250000                        
           num_steps_stop: 80000                         
              num_workers: 4                             
                    power: 0.9                           
               print_freq: 100                           
             restore_from: None                          
          save_pred_every: 10000                         
                      set: train                         
             snapshot_dir: ./snapshots/syn2city          
                   source: synthia                       
                   target: cityscapes                    
             weight_decay: 0.0005                        
----------------- End -------------------
Result are shown below: 
===>road:	1.41
===>sidewalk:	2.32
===>building:	23.02
===>wall:	0.0
===>fence:	0.01
===>pole:	2.71
===>light:	0.86
===>sign:	1.33
===>vegetation:	8.22
===>terrain:	0.0
===>sky:	63.95
===>person:	15.73
===>rider:	0.26
===>car:	0.75
===>truck:	0.0
===>bus:	0.07
===>train:	0.0
===>motocycle:	0.02
===>bicycle:	0.03
===> mIoU: 6.35

**Comment by liyunsheng13** [comment]
You use the hyper parameters for DeepLab ResNet101 to train VGG16. I suggest you to first test the model I upload to make sure everything is correct.

**Comment by viyjy** [comment]
> You use the hyper parameters for DeepLab ResNet101 to train VGG16. I suggest you to first test the model I upload to make sure everything is correct.

Yunsheng, would you please provide the hyper parameters for VGG16? Thanks.

**Comment by JiangXiaobai00** [comment]
Hi, I can't open the url on drive.google. Is there any other path to download this data set？
